server/schedulers/balance_leader.go

```diff
@@ -161,48 +170,70 @@ func (l *balanceLeaderScheduler) Schedule(cluster schedule.Cluster) []*operator.
 			targets[j].LeaderScore(leaderSchedulePolicy, jOp)
 	})
 
+	usedStores := make(map[uint64]struct{})
```

Comment:
I think the stores can be used more than once.

Reply:
has changed